src/tests/cmocka/common_utils.c

```diff
@@ -0,0 +1,66 @@
+/*
+    Authors:
+        Sumit Bose <sbose@redhat.com>
+
+    Copyright (C) 2022 Red Hat
+
+    SSSD tests: common helpers
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "tests/cmocka/common_mock.h"
+
+static struct CMUnitTest select_test(const struct CMUnitTest tests[],
+                                     size_t test_count, const char *name)
+{
+    size_t c;
+    struct CMUnitTest empty = { 0 };
+
+    for (c = 0; c < test_count; c++) {
+        if (strcmp(tests[c].name, name) == 0) {
+            return tests[c];
+        }
+    }
+
+    return empty;
+}
+
+void list_tests(FILE *file, const char *pref,
```

Comment:
This function is unused. Did you forget to call it?

Reply:
It is called if the test is called with the new `--list-tests` command line option.

Reply:
I didn't notice that. Thanks for the explanation.